troposphere/templates/login.html

```diff
@@ -84,7 +83,7 @@
                 {% csrf_token %}
                 {% if disable_login == False %}
                 <a class="submitButton" href="{{ BASE_URL }}/login">
-                        Log in with your iPlant ID
+                        Log in with your {{ ORG_NAME }} ID
```

Comment:
You are right -- I was thinking about JetStream when I wrote this up, and in that case "Log in to JetStream" would be 'correct' given the situation.

That may/may not hold for other themes like CyVerse. Wording should change, but what it changes to is clearly up in the air.